helm-charts/common/teirerank/gaudi-values.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for teirerank.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+accelDevice: "gaudi"
+
+image:
+  repository: opea/tei-gaudi
+  tag: "1.0"
```

Comment:
For all opea images, we use "latest" tag in the main branch, and change it later in the release-rc branch. Please use "latest" here.

Reply:
done